spreadsheets/scalerank/landuse.csv

```diff
@@ -1 +1,72 @@
-kind,scalerank
+kind,operator,zoom::int,area::int,output:tier::int,output:scalerank::int
+national_park;park or protected land;battlefield,!United States Forest Service,>=3,>=300000000,1,3
+national_park;park or protected land;battlefield,!United States Forest Service,>=4,>=300000000,1,4
+national_park;park or protected land;battlefield,!United States Forest Service,>=5,>=150000000,1,5
+national_park;park or protected land;battlefield,!United States Forest Service,>=6,>=150000000,1,6
+national_park;park or protected land;battlefield,!United States Forest Service,>=7,>=100000000,1,7
+national_park;park or protected land;battlefield,!United States Forest Service,>=8,>=10000000,1,8
+national_park;park or protected land;battlefield,!United States Forest Service,>=9,>=5000000,1,9
+national_park;park or protected land;battlefield,!United States Forest Service,>=10,>=1000000,1,10
+national_park;park or protected land;battlefield,!United States Forest Service,>=11,>=500000,1,11
+national_park;park or protected land;battlefield,!United States Forest Service,>=12,>=500000,1,12
+national_park;park or protected land;battlefield,!United States Forest Service,>=13,>=100000,1,13
+national_park;park or protected land;battlefield,!United States Forest Service,>=14,>=50000,1,14
+national_park;park or protected land;battlefield,!United States Forest Service,>=15,>=2000,1,15
+national_park;park or protected land;battlefield,!United States Forest Service,>=16,+,1,16
+national_park;park or protected land,!United States National Park Service,>=4,>=1000000000,2,4
+national_park;park or protected land,!United States National Park Service,>=5,>=1000000000,2,5
+national_park;park or protected land,!United States National Park Service,>=6,>=150000000,2,6
+national_park;park or protected land,!United States National Park Service,>=7,>=100000000,2,7
+national_park;park or protected land,!United States National Park Service,>=8,>=10000000,2,8
+national_park;park or protected land,!United States National Park Service,>=9,>=5000000,2,9
+national_park;park or protected land,!United States National Park Service,>=10,>=1000000,2,10
+national_park;park or protected land,!United States National Park Service,>=11,>=500000,2,11
+national_park;park or protected land,!United States National Park Service,>=12,>=250000,2,12
+national_park;park or protected land,!United States National Park Service,>=13,>=100000,2,13
+national_park;park or protected land,!United States National Park Service,>=14,>=50000,2,14
+national_park;park or protected land,!United States National Park Service,>=15,>=2000,2,15
+national_park;park or protected land,!United States National Park Service,>=16,+,2,16
+conservation;protected_area;nature_reserve;urban;rural;residential,,>=4,>=1000000000,2,4
```

Comment:
Paired with https://github.com/tilezen/vector-datasource/pull/980/files#r76012452, to make this work better outside of United States the filter needs to be expanded to include kinds of `national_park` and NOT several `protected_class` values:

```
            conservation:
                filter:
                    all:
                        - $zoom: { min: 4 }
                        - kind: [conservation, protected_area, nature_reserve, national_park]
                        - not: { protect_class: ['2','3','5'] }
```